project/V.scala

```diff
@@ -12,6 +12,8 @@ object V {
   val zioKafkaVersion            = "0.17.1"
   val zio2KafkaVersion           = "2.0.0"
   val zioHttpVersion             = "0.0.4"
+  val zioAwsVersion              = "5.20.17.1"
```

Comment:
I believe this is the latest version of `zio-aws` that uses ZIO `2.0.9`. Usage of more recent versions causes `NoClassDefFoundError`. It should be updated after the ZIO version is bumped up.